spaces/bella/skin.scss

```
@@ -0,0 +1,9 @@
+.name {
```

This must be the id of your space
```suggestion
.bella {
```